### PR TITLE
LPS-116937 | master

### PR DESCRIPTION
**Pull request by Tim-Cao**
https://issues.liferay.com/browse/LPS-116937

This is only for master.

@jkappler

### CODE REVIEW

**Comment by Tim-Cao** [comment]
ci:test:sf

**Comment by Tim-Cao** [comment]
ci:test:sf

**Comment by Tim-Cao** [comment]
ci:test:sf

**Comment by Tim-Cao** [comment]
ci:test:sf

**Comment by Tim-Cao** [comment]
ci:test:sf

**Comment by Tim-Cao** [comment]
ci:test:sf

**Comment by Tim-Cao** [comment]
ci:test:sf

**Comment by Tim-Cao** [comment]
ci:test:sf

File: modules/apps/layout/layout-page-template-test/src/testIntegration/java/com/liferay/layout/page/template/service/test/LayoutPageTemplateEntryServiceTest.java [hunk 1]
```diff
@@ -209,6 +229,89 @@ public void testAddMultipleLayoutPageTemplateEntries()
...
+
+		Assert.assertEquals(
+			layoutPageTemplateEntry.getName(),
+			persistedLayoutPageTemplateEntry.getName());
+
+		Assert.assertNotNull(layoutPageTemplateEntry.getLayoutPrototypeId());
+	}
+
+	@Test
+	public void testAssertGroupOfWidgetLayoutPageTemplateEntry()
```
Comment (jkappler) [inline modules/apps/layout/layout-page-template-test/src/testIntegration/java/com/liferay/layout/page/template/service/test/LayoutPageTemplateEntryServiceTest.java:280]: @Tim-Cao, this assertion can be done in the previous test. We could check if the layout was created, if the layout prototype was created, and if it's the correct group

File: modules/apps/layout/layout-page-template-test/src/testIntegration/java/com/liferay/layout/page/template/service/test/LayoutPageTemplateEntryServiceTest.java [hunk 2]
```diff
@@ -209,6 +229,89 @@ public void testAddMultipleLayoutPageTemplateEntries()
...
+		LayoutPageTemplateEntry layoutPageTemplateEntry =
+			LayoutPageTemplateTestUtil.addWidgetLayoutPageTemplateEntry(
+				_layoutPageTemplateCollection.
+					getLayoutPageTemplateCollectionId());
+
+		LayoutPrototype persistedLayoutPrototype =
+			_layoutPrototypeLocalService.getLayoutPrototype(
+				layoutPageTemplateEntry.getLayoutPrototypeId());
+
+		LayoutPrototype layoutPrototype =
```
Comment (jkappler) [inline modules/apps/layout/layout-page-template-test/src/testIntegration/java/com/liferay/layout/page/template/service/test/LayoutPageTemplateEntryServiceTest.java:307]: @Tim-Cao, we are comparing the same object here. What we could do is compare the names of the layoutPageTemplateEntry and the name of the LayoutPrototype (they should be the same)

**Comment by jkappler** [comment]
@Tim-Cao, it's a really nice pull 👍. I was thinking that we could add also another test to check if the layoutPrototype is deleted when a LayoutPageTemplateEntry is also deleted. What do you think?

File: modules/apps/layout/layout-page-template-test/src/testIntegration/java/com/liferay/layout/page/template/service/test/LayoutPageTemplateEntryServiceTest.java (hunk 1 above)
Comment (jkappler) [inline modules/apps/layout/layout-page-template-test/src/testIntegration/java/com/liferay/layout/page/template/service/test/LayoutPageTemplateEntryServiceTest.java:280]: @Tim-Cao, I thing that for now we can leave this out, until https://issues.liferay.com/browse/LPS-115365 is fixed. Then we can add more tests for the group (even though I mentioned to create this test). What we can do for the moment is move the assert to the previous test. What do you think?

**Comment by Tim-Cao** [comment]
ci:close

**Comment by Tim-Cao** [comment]
See https://github.com/jkappler/liferay-portal/pull/856

**Pull request closed without merging**